### ISSUE TITLE
Colors for light theme

### ISSUE THREAD

**Issue by soviet-man** [ogham/exa#1071]
Can you suggest set of colors for light system theme? 

I prefer light theme, and colors designed for dark terminal are not comfortable.

![изображение](https://user-images.githubusercontent.com/33825590/169584382-c3b3fda3-0038-4346-8ebf-c40b829889a8.png)

**Comment by Boberro** [comment]
Actually, dark theme is no better, "date modified" has wrong contrast as well.
<img width="470" alt="image" src="https://user-images.githubusercontent.com/7759852/169808573-0224d291-ae5e-453f-b913-47dbc5623053.png">

**Comment by soviet-man** [comment]
> Actually, dark theme is no better, "date modified" has wrong contrast as well. <img alt="image" width="470" src="https://user-images.githubusercontent.com/7759852/169808573-0224d291-ae5e-453f-b913-47dbc5623053.png">

How to set colors only for "Name" column?

**Comment by ariasuni** [comment]
I don’t think you can change column colors as of right now. I’m compiling my own version of exa with #370 that I may merge if @ogham doesn’t come back soon, since I think it’s a good change, but it would also be nice to be able to customize that.

**Comment by danilobjr** [comment]
@iandoug Same here. 

I've tried `da` code as you can see in [docs](https://the.exa.website/docs/colour-themes), but no effects.
